wiki/Tournaments/GBC/GBC_2023_Spring/zh.md

```diff
@@ -0,0 +1,219 @@
+| 解说 | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
+| 转播 | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [aiyulu](https://osu.ppy.sh/users/189617), ::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273) |
+| 数据统计 & wiki 编写 | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308), ::{ flag=CN }:: [FAIRYBYUN](https://osu.ppy.sh/users/17575228) |
+
+## 链接
+
+- [论坛讨论帖](https://osu.ppy.sh/community/forums/topics/1730051)
+- 直播间
+  - [主舞台](https://live.bilibili.com/22545296) (::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407))
+  - [副舞台](https://live.bilibili.com/10183) (::{ flag=CN }:: [aiyulu](https://osu.ppy.sh/users/189617))
+  - [副舞台](https://live.bilibili.com/6605693) (::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273))
+- [比赛 QQ 群](https://jq.qq.com/?_wv=1027&k=zqBeTqMW)
+- [赛事信息表格](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY?tab=uitssi)
+
+## 参赛选手
+
+| 组别 | 选手 |
+| :-: | :-- |
+| EX 组 | Pending |
+| S 组 | Pending |
+| A 组 | Pending |
+| B 组 | Pending |
+
+## 赛果
+
+由于选手非常多，将使用 **[赛事信息表格](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY)** 呈现赛果。
+
+## 规则
+
+### 常规规则
+
+1. GB 杯 2023 春季赛是周常积分个人赛事，使用 osu!mania 游戏模式。
+   - 选手获得的积分根据图池总排名换算。
+   - 图池总排名根据单曲排名汇总得到。
+2. 谱面计分方式采用 `Score V2`。
+3. GB 杯 2023 春季赛共设置 4 个组别，每周仅进行 2 个组别的比赛。
+4. 主办方将会在比赛开始前的上个周日于主舞台发布图池。
+   - 周常赛中每一轮图池包含 `7` 张谱面，键型并不固定；
+5. 选手在本周比赛开始前选择比赛场次。
+   - 有特殊时间要求的玩家请提前联系裁判商定特殊时间场次。主办方并不保证满足任何人的特殊时间要求，请自行提前与裁判妥善沟通；
+6. 每周，裁判会在固定时间创建房间，想要出分的选手进入房间准备进行谱面出分。
+7. 比赛**没有**热手阶段。
+8. 允许使用皮肤或者修改显示设定禁用叙事板及打击音效，但皮肤**不可以**修改核心游戏机制。
+
+### 比赛报名
+
+1. 请填写 **[报名表](https://wj.qq.com/s2/11825634/aba7/)** 来进行报名。
+   - 报名后的选手**必须**加入 [赛事官方 QQ 群](https://jq.qq.com/?_wv=1027&k=zqBeTqMW)。
+   - 如果存在无法使用QQ的情况请通过 osu! 私信或 Discord 联系主办方并使用 Discord 进行交流。
+2. 主办方会手动审核所有报名选手来确保可行性和严肃性。
+   - **本届赛事没有排名限制**。
+   - 选手不得在 12 个月内有违反 **[osu! 社区规则](/wiki/Rules)** 的记录。
+   - 报名时有软性 `Play counts` 要求，重度 `deranker` 和 `炸鱼` 选手可能会被强制升级。
+3. 报名结束后，主办方会公布所有成功报名玩家的名单与对应分组。分组方式如下：
+
+| 组别 | 选手的4kPP |
+| EX | 8500 and above |
+| S | 6500 ~ 8499 |
+| A | 4500 ~ 6499 |
+| B | 0 ~ 4499 |
+
+4. 如果选手的4kpp超过其组别的上限过多，将会被强制升级到更高的组别。缓冲区为100pp
```

Comment:
"4kpp" can be misinterpreted as 4,000 pp instead of "pp in osu!mania 4K"

Reply:
It's nearly impossible to consider "4kpp" as "4,000" pp in Chinese scene, "4键pp" is rarely used , even in Osu website, it's using 4k instead of 4键
![image](https://user-images.githubusercontent.com/20500319/231621116-9eb123ac-2ab0-4102-a270-2a6c9aeffdce.png)

Reply:
As a native speaker, it looks nothing wrong for me :)